editorextensions/codebeamer-custom-ui/modal/src/pages/import/components/query/cbqlToggle/CbqlToggle.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { setAdvancedSearch } from '../../../../../store/slices/userSettingsSlice';
+import { RootState } from '../../../../../store/store';
+import { OverlayTrigger, Tooltip } from 'react-bootstrap';
+
+export default function CbqlToggle() {
+	const dispatch = useDispatch();
+	const { advancedSearch } = useSelector(
+		(state: RootState) => state.userSettings
+	);
+
+	const toggleSearchMethod = () => {
+		dispatch(setAdvancedSearch(!advancedSearch));
+	};
+
+	return (
+		<DefaultOverlayTrigger
+			content={advancedSearch ? 'Assisted Query' : 'CBQL Input'}
+		>
+			<button
+				className={`mx-1 
+							${
+								advancedSearch
+									? 'button-icon-small button-icon button-icon-secondary icon-parameters'
+									: 'button button-secondary button-small'
+							}`}
+				onClick={toggleSearchMethod}
+				data-test="search-method"
+			>
+				{advancedSearch ? (
```

Comment:
I'd argue the expression here should be inverted. You want  ![image](https://github.com/codeBeamer-Extensions-and-Addons/codebeamer-lucidspark/assets/70019423/f1828778-9ff7-4727-a08c-b10b496c78c0) to show when you're on the assisted query, to _switch to CBQL_, and vice-versa. You don't need "confirmation" you're right now on the CBQL input, but to be informed of the alternative.